From c9d5cc91f0191b3006ff693718fe2f57b63460c0 Mon Sep 17 00:00:00 2001
From: Allen Xu <b45815@freescale.com>
Date: Sat, 22 Nov 2014 13:15:19 -0600
Subject: [PATCH 1166/1691] MLK-9909 ARM: dts: add imx6sx sdb revA QSPI legacy
 support

commit c9d5cc91f0191b3006ff693718fe2f57b63460c0 from
git://git.freescale.com/imx/linux-2.6-imx.git imx_4.1.15_1.0.0_ga

add sdb revA board QSPI legacy support for Spansion QSPI chip.

Signed-off-by: Allen Xu <b45815@freescale.com>
(cherry picked and merge from commit 3fd143ae78007e58d1dc45bc829b462a12b93cd2)
(cherry picked from commit 8317ae5998a20b3303a2a9209c29ab5674e09dc3)
---
 arch/arm/boot/dts/imx6sx-sdb.dts | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm/boot/dts/imx6sx-sdb.dts b/arch/arm/boot/dts/imx6sx-sdb.dts
index 1a3dd28..39d9ba3 100644
--- a/arch/arm/boot/dts/imx6sx-sdb.dts
+++ b/arch/arm/boot/dts/imx6sx-sdb.dts
@@ -37,6 +37,7 @@
 	pinctrl-0 = <&pinctrl_qspi2>;
 	status = "okay";
 
+#ifndef SPANSIONFLASH
 	flash0: n25q256a@0 {
 		#address-cells = <1>;
 		#size-cells = <1>;
@@ -52,6 +53,7 @@
 		spi-max-frequency = <29000000>;
 		reg = <1>;
 	};
+#endif
 };
 
 &usbphy1 {
-- 
1.9.1

